### PR TITLE
Week3 extra

### PR DESCRIPTION
**Pull request by LauraOD1211**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -89,4 +93,60 @@ function deleteComments () {
...
   }
+}
+
+/**
+ * Updates and displays votes
+ */
+function updateVotes (id, vote) {
+  switch (vote) {
+    case "up":
+      if (document.getElementById(id).classList.contains("unvoted")) {
```
Comment (chripell) [inline portfolio/src/main/webapp/script.js]: I don't understand the logic behind the various if/else cases, can you please add a comment?

**Review by chripell — APPROVED** [review]
As I mentioned in the inline comment, I don't understand the logic about updating display with up/down votes. Can you add a comment please?

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (LauraOD1211) [reply portfolio/src/main/webapp/script.js]: Will do. I'm in the middle of improving this at the minute, hoping it'll be clearer by the time I'm finished :)

**Review by chripell — APPROVED** [review]
Thanks for the clarification comments!

**Pull request merged**